### ISSUE TITLE
Class inheritance and "prefix with I" option

### ISSUE THREAD

**Issue by mailivore** [Evertras/typescript-cs-poco#22]
Hi, thanks for your generator, it is soooooo nice but I have a small problem.

Let's consider I would create an abstract class that will be used as a base for all of my other classes : 

```
public abstract class Base
{
        public bool deleted { get; set; }
}

public class Meeting : Base
{
        public DateTime Day { get; set; }
}
```

This would give me the following result : 
```
export interface IBase {
        deleted: bool;
}
export interface IMeeting extends Base {
        Day : string;
}
```

The problem is that the generated IMeeting extends the "Base" class which is not defined but "IBase" is.

One workaround would be to disable the "prefix with I" option but I think this is a nice feature.

Could you fix the name of the extended class when the "prefix with I" option is used?

Thanks.

**Comment by Evertras** [comment]
That absolutely needs a fix.  I'll get to that when I get a chance!

**Comment by Evertras** [comment]
Hey there mailivore, I'm really, really sorry this took so long.  Work has been insane for the last month.  I've made the fix and it's available in 1.7.3, you should get `IBase` instead of `Base` as desired.
